geomesa-plugin/src/main/resources/META-INF/services/org.geotools.data.DataStoreFactorySpi

```diff
@@ -1 +1 @@
-accumulo.data.AccumuloDataStoreFactory
+org.locationtech.geomesa.accumulo.data.AccumuloDataStoreFactory
```

Comment:
i think this file is actually redundant and can be completely removed, as it comes in already from the geomesa-accumulo module.